docs/UserGuide.md

```diff
@@ -216,15 +216,15 @@ Format: `help`
 
 ##### Clearing all entries : `clear`
 
-Clears all entries from the address book.
+Clears all entries from the address book and appointment book after a validation check from the user.
 
 Format: `clear`
 
 <br>
 
 ##### Exiting the program : `exit`
 
-Exits the program.
+Exits the program after printing a message.
```

Comment:
Should we specify that the the message is logged in the terminal, since it is not shown to the user in the GUI?